operator/pkg/ciliumendpointbatch/manager_test.go

```diff
@@ -175,6 +175,27 @@ func TestDeepCopyCeps(t *testing.T) {
 		m.updateCebInCache(Ceb, true)
 		ceb, _ = m.getCebFromCache(cn)
 		assert.Equal(t, ceb.DeepEqual(Ceb), true, "Local CEB should match with CEB in datastore")
+
+		m1 := newCebManagerIdentity(newQueue(), 2)
+		m1.InsertCepInCache(cep1)
+		cn = m1.InsertCepInCache(cep2)
+		ceb, _ = m1.getCebFromCache(cn)
+		assert.Equal(t, m1.getCebCount(), 2, "Total number of CEBs allocated is 2")
+		assert.Equal(t, m1.getTotalCepCount(), 2, "Total number of CEPs inserted is 2")
+		assert.Equal(t, len(ceb.Endpoints), 1, "Total number of CEPs inserted is 1")
+
```

Comment:
Current test does not cover inserting CEPs of same identity and assert they end up in same CEB, or am I missing it?

Reply:
Update test case to cover the case you had mentioned.